guarddog/cli.py

```diff
@@ -165,14 +165,18 @@ def _scan(identifier, version, rules, exclude_rules, output_format, exit_non_zer
     if scanner is None:
         sys.stderr.write(f"Command scan is not supported for ecosystem {ecosystem}")
         exit(1)
-    results = {}
+    results = []
     if is_local_target(identifier):
         log.debug(f"Considering that '{identifier}' is a local target, scanning filesystem")
-        results = scanner.scan_local(identifier, rule_param)
+        if os.path.isdir(identifier):
```

Comment:
this is a new feature (ability to scan a folder of packages), can we add an example to the README?

Reply:
Done